### ISSUE TITLE
Submitting orders without payment info

### ISSUE THREAD

**Issue by chrischen** [printdotio/printio-api#9]
I have a credit card on file with you guys. How do I submit the API call to place an order without submitting payment info?

**Comment by micahasmith** [comment]
Its pretty easy. All you need to do is in the `Payment` object submit your `PartnerBillingKey`.

Note that you do not need to track and sum and submit the order/item totals when the item is on credit.

See: https://api.print.io/docs/#!/orders/POST_post_1 and http://print.io/api

**Comment by chrischen** [comment]
Where do I get the partnerbillingkey?

**Comment by micahasmith** [comment]
Sometimes the sales reps give it as the `API_KEY`, did you receive this?

**Comment by chrischen** [comment]
Yes. I received an API_KEY and recipe id.

**Comment by micahasmith** [comment]
Use the API_KEY for that value. You'll be all good.

**Comment by chrischen** [comment]
I used the API key and submitted in Payment object as PartnerBillingKey.

I get back:
{u'HadError': True, u'Errors': [{u'PropertyName': u'Partner Billing Key', u'ErrorMessage': u'Invalid key', u'AttemptedValue': None, u'CustomState': None}], u'ErrorReferenceCode': u'0274071b-c167-48ca-b920-a117dfbde16c'}

EDIT: I didn't add item total.
EDIT: Added Total value but same error. Does it give that error if the total amount is not correct?
